src/Controller/Component/ModulesComponent.php

```diff
@@ -68,8 +68,9 @@ class ModulesComponent extends Component
      */
     public function startup(): void
     {
+        /** @var \Authentication\Identity $user */
```

Comment:
I see a lot of these annotations in this PR. What's the type that is automatically inferred by phpstan?

I think that in some cases the correct type to document would be `\Authentication\Identity|null`, which explains why there used to be a few `empty($user)` checks…

Reply:
The inferred type was the interface. Good point on using `\Authentication\Identity|null` instead of `Authentication\Identity`